sdk/frontdoor/arm-frontdoor/package.json

```diff
@@ -28,14 +28,14 @@
   "module": "./dist-esm/src/index.js",
   "types": "./types/arm-frontdoor.d.ts",
   "devDependencies": {
-    "@microsoft/api-extractor": "^7.18.11",
```

Comment:
only need to keep this line from the main branch, the other part doesn't need to use main branch